### ISSUE TITLE
Feature Request: expose command line in object model

### ISSUE THREAD

**Issue by tobyhodges** [common-workflow-language/common-workflow-language#468]
WIP
expose commandline in object model
  - suggested syntax: arguments: [array, of, strings]
  either pass an index or a name (using optional name field to arguments)

**Comment by tetron** [comment]
There's already an `arguments` field:

http://www.commonwl.org/v1.0/CommandLineTool.html#CommandLineTool

Use it this way:

```
inputs:
  inp1: File
arguments: [mycommand, $(inputs.inp1)]
```

**Comment by mr-c** [comment]
@tetron we updated the description to be clearer

**Comment by tetron** [comment]
Ok, that makes more sense now.  The main question is probably order of operations, this can only be made available for expressions evaluated after the command line is constructed (output, mainly).

**Comment by mr-c** [comment]
Yep! The intended usage is in `outputBinding`s (soon to be `results:`) so that shouldn't be a problem but will be good to specify.
